slo/src/Utils.cs

```diff
@@ -0,0 +1,12 @@
+using Ydb.Sdk;
+
+namespace slo;
+
+public static class Utils
+{
+    public static string GetResonseStatusName(StatusCode statusCode)
+    {
+        var prefix = statusCode >= StatusCode.ClientTransportResourceExhausted ? "GRPC" : "YDB";
```

Comment:
Why the StatusCode.ClientTransportResourceExhausted used as separator between grpc/ydb codes?